app/features/history/history_filter/EditHistoryFilterView.js

```diff
@@ -0,0 +1,462 @@
+import React, { Component } from 'react';
+import {
+    Text,
+    TextInput,
+    StyleSheet,
+    View,
+    TouchableOpacity,
+    Platform,
+    TouchableHighlight,
+    Image,
+    Modal,
+    StatusBar,
+} from 'react-native';
+import { Slider } from 'react-native';
+import { SETTINGS_PANEL_STYLES } from 'app/appearance/styles/GlobalStyles';
+import EditHistoryFilterTagsToIncludeScreen from './tags/tagsToInclude/EditHistoryFilterTagsToIncludeScreen';
+import EditHistoryFilterTagsToExcludeScreen from './tags/tagsToExclude/EditHistoryFilterTagsToExcludeScreen';
+import EditHistoryExerciseScreen from './exercise_name/EditHistoryFilterExerciseScreen';
+import EditFilterStartDateScreen from './dateRange/startDate/EditHistoryFilterStartScreen';
+import EditFilterEndDateScreen from './dateRange/endDate/EditHistoryFilterEndScreen';
+import Pill from 'app/shared_features/pill/Pill';
+import * as Device from 'app/utility/Device';
+import Icon from 'react-native-vector-icons/FontAwesome';
+import EditHistoryFilterStartDateScreen from './dateRange/startDate/EditHistoryFilterStartScreen';
+import EditHistoryFilterEndDateScreen from './dateRange/endDate/EditHistoryFilterEndScreen';
+
+class EditHistoryFilterView extends Component {
+
+    _close() {
+        this.props.closeModal();
+    }
+
+    // RENDER
+
+    _displayTagsToInclude() {
```

Comment:
it might've been my original bad names, but having both display and render isn't descriptiive at all of what's going on.

Not the biggest deal though, letting it through. Ideally it would be like render the pills and render the entire section